CHANGES.md

```diff
@@ -0,0 +1,22 @@
+# HBASE Changelog
```

Comment:
nit: Copyright at the beginning of this file? Same to `RELEASENOTES.md`

Reply:
Thanks for the review.

I can't put copyright/license at head of the flie because it messes up markdown viewers.

Reply:
We render these to html, so aren't standard xml `<!-- -->` applicable? I guess it would be as noise for anyone opening the file locally though.

Reply:
May need revisiting. In old days, rendering this markdown in browser would hide the license w/ these markings. After a recent experience, trying to read from filesystem using various markdown viewers, may need more work.